boards/shields/esp_8266/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+#if SHIELD_ESP_8266
+
+#if NETWORKING
```

Comment:
I suppose commenting out the `if`s wasn't your intention here?

Reply:
Fixed!